CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## v1.5.0
 
+- [security] updated version range of `decompress` to fix the known [security vulnerability](https://snyk.io/vuln/npm:decompress) [#8924](https://github.com/eclipse-theia/theia/pull/8294)
```

Comment:
I think this link to the specific vulnerability would be better: https://snyk.io/vuln/SNYK-JS-DECOMPRESS-557358

Reply:
Done :)